tests/deployment/ipmi.py

```diff
@@ -86,6 +86,9 @@ def test_node_set_bootdev(self):
                 api.node_set_bootdev(node.label, 'invalid-device')
 
         # register a node with erroneous ipmi details to raise OBMError
+        # XXX: In theory, this could actually be a real node; we should take
+        # some measure to ensure this never collides with something actually
```

Comment:
Yeah I figured we could do something like that; I just wanted to document the problem while I was in there.